gui/static/js/components/controlpanel.js

```diff
@@ -27,21 +27,42 @@ export default class ControlPanel extends React.Component {
 
     }
 
+
+     /* sends a command to bot according to the button pressed */
     onKeyDown(event){
         if (this.state.keyboard){
-            if (event.key == 'w'){
-                this.sendMotors(100, 100, 100, 100);
+           if(event.keyCode==87) {
+                // If the 'W' key is pressed,move forward
+                this.sendMotors(20,20,20,20);
+                var a=this;
+                setTimeout(function(){
```

Comment:
When I test this on the bots, they move very erratically. I think we should model off the old code (from cs-minibot-platform), here (https://github.com/cornell-cup/cs-minibot-platform/blob/develop/cs-minibot-platform-src/src/main/resources/public/js/interface.js#L421 ). Here and below.

Reply:
ok. But I don't think it's codes' problem. You can test press button go forward/back/left/right button on gui on the new bot, the bot not goes to the right direction. I've tested in Thursday on .170 one, which works.